### PR TITLE
chore: hsl-update-colors

### PR DESCRIPTION
**Pull request by sean-brydon**
Updates our colours to use HSL instead of hex - allows us to take advanage of bg-opacity a bit easier than previosuly

### CODE REVIEW

**Comment by sean-brydon** [comment]
Current dependencies on/for this PR:
* main
  * **PR #12199** <a href="https://app.graphite.dev/github/pr/calcom/cal.com/12199" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/calcom/cal.com/12199?utm_source=stack-comment).

File: packages/features/shell/Shell.tsx [hunk 1]
```diff
@@ -828,7 +828,7 @@ function SideBar({ bannersHeight, user }: SideBarProps) {
     <div className="relative">
       <aside
         style={{ maxHeight: `calc(100vh - ${bannersHeight}px)`, top: `${bannersHeight}px` }}
-        className="desktop-transparent bg-muted border-muted fixed left-0 hidden h-full max-h-screen w-14 flex-col overflow-y-auto overflow-x-hidden border-r dark:bg-gradient-to-tr dark:from-[#2a2a2a] dark:to-[#1c1c1c] md:sticky md:flex lg:w-56 lg:px-3">
```
Comment (sean-brydon) [inline packages/features/shell/Shell.tsx:831]: Matching designs

**Comment by sean-brydon** [comment]
@ciaranha Do you think you can checkout the preview URL of this and confirm colours are right. I know we discussed the bg on login/signup being not what you expected

**Comment by PeerRich** [comment]
LGTM whats needed to leave draft?

**Review by PeerRich — DISMISSED** [review]

**Review by PeerRich — APPROVED** [review]
looks good to me. havent tested all colors but i trust you

**Comment by Udit-takkar** [comment]
@sean-brydon the colors are different now.

PRODUCTION:-

Screenshot 2023-12-15 at 6 30 52 PM


LATEST MAIN:-
Screenshot 2023-12-15 at 6 30 08 PM

**Comment by sean-brydon** [comment]
> @sean-brydon the colors are different now.
> 
> PRODUCTION:-
> 
> Screenshot 2023-12-15 at 6 30 52 PM
> LATEST MAIN:- Screenshot 2023-12-15 at 6 30 08 PM

Me and @ciaranha did this live on a call together - Ciaran can you just confirm this is as expected

**Pull request merged**